content/docs/06-integrations/00-ubuntu.md

```diff
@@ -35,26 +58,124 @@ Ubuntu is a Linux distribution based on Debian and composed mostly of free and o
 
-</Tabs.TabPane>
+<br />
+<br />
 
-</Tabs>
+```yml
+ kubeadmconfig:
+  preKubeadmCommands:
+  - echo "Executing pre kube admin config commands"
+  - update-ca-certificates
+  - 'systemctl restart containerd; sleep 3'
+  - 'while [ ! -S /var/run/containerd/containerd.sock ]; do echo "Waiting for containerd..."; sleep 1; done'
+  postKubeadmCommands:
+  - echo "Executing post kube admin config commands"
+  files:
+  - targetPath: /usr/local/share/ca-certificates/mycom.crt
+    targetOwner: "root:root"
+    targetPermissions: "0644""
+    content: |
+      -----BEGIN CERTIFICATE-----
+      MIICyzCCAbOgAwIBAgIBADANBgkqhkiG9w0BAQsFADAVMRMwEQYDVQQDEwprdWJl
+      cm5ldGVzMB4XDTIwMDkyMjIzNDMyM1oXDTMwMDkyMDIzNDgyM1owFTETMBEGA1UE
+      AxMKa3ViZXJuZXRlczCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMdA
+      nZYs1el/6f9PgV/aO9mzy7MvqaZoFnqO7Qi4LZfYzixLYmMUzi+h8/RLPFIoYLiz
+      qiDn+P8c9I1uxB6UqGrBt7dkXfjrUZPs0JXEOX9U/6GFXL5C+n3AUlAxNCS5jobN
+      fbLt7DH3WoT6tLcQefTta2K+9S7zJKcIgLmBlPNDijwcQsbenSwDSlSLkGz8v6N2
+      7SEYNCV542lbYwn42kbcEq2pzzAaCqa5uEPsR9y+uzUiJpv5tDHUdjbFT8tme3vL
+      9EdCPODkqtMJtCvz0hqd5SxkfeC2L+ypaiHIxbwbWe7GtliROvz9bClIeGY7gFBK
+      jZqpLdbBVjo0NZBTJFUCAwEAAaMmMCQwDgYDVR0PAQH/BAQDAgKkMBIGA1UdEwEB
+      /wQIMAYBAf8CAQAwDQYJKoZIhvcNAQELBQADggEBADIKoE0P+aVJGV9LWGLiOhki
+      HFv/vPPAQ2MPk02rLjWzCaNrXD7aPPgT/1uDMYMHD36u8rYyf4qPtB8S5REWBM/Y
+      g8uhnpa/tGsaqO8LOFj6zsInKrsXSbE6YMY6+A8qvv5lPWpJfrcCVEo2zOj7WGoJ
+      ixi4B3fFNI+wih8/+p4xW+n3fvgqVYHJ3zo8aRLXbXwztp00lXurXUyR8EZxyR+6
+      b+IDLmHPEGsY9KOZ9VLLPcPhx5FR9njFyXvDKmjUMJJgUpRkmsuU1mCFC+OHhj56
+      IkLaSJf6z/p2a3YjTxvHNCqFMLbJ2FvJwYCRzsoT2wm2oulnUAMWPI10vdVM+Nc=
+      -----END CERTIFICATE-----
+```
+
+# Ubuntu Advantage
+Canonical Ubuntu Advantage extends your infrastructure's security, certified compliance, and 24x7 support. The Ubuntu Advantage (UA) service is available only for the MAAS cloud. With Palette, enable the UA services when modifying the Pack Value. See below for steps on how to modify the Preset options.
+
+**Benefits with UA**:
+
+<br />
+
+#### Enterprise 
+
+- Extended Security Maintenance
+- Kernel Livepatch service to avoid reboots
+- FIPS 140-2 Level 1 certified crypto modules
+- Common Criteria EAL2
+
+<br />
+
+#### Personal Use
+
+- Free Personal Token for up to three machines with UAI.
+
+For more information see [Ubuntu Advantage for Infrastructure](https://ubuntu.com/advantage) site.
+
+## Modifying the Presets for MAAS Cloud
+
+1. Palette allows you to include the Ubuntu Advantage service in the **Profile Layers** section, when you create a new cluster profile. 
+
+
+2. Give the new Pack a **Name**, **Version number**, **Description**, **Type**, and **Tags** and click the **Next** button.
+
+
+3. Select **MAAS** as the Infrastructure provider and click the **Next** button.

```

Comment:
Not just MAAS

Reply:
Okay removed "MAAS" and added "your cloud provider".